testsuite/smoke-test/src/state_sync_v2.rs

```diff
@@ -231,6 +254,26 @@ async fn test_validator_bootstrap_transactions() {
     test_validator_sync(swarm).await;
 }
 
+#[tokio::test]
+async fn test_validator_bootstrap_transactions_no_compression() {
```

Comment:
Do you want to explicitly enabled compression in `test_full_node_bootstrap_transactions`, so that even if we turn off compression by default, we are still testing both compressed and uncompressed path? Also, possible to reuse the code with above test? :)

Reply:
- Cool, I've updated `test_full_node_bootstrap_transactions` and `test_full_node_bootstrap_outputs` to explicitly enable it.
- Regarding re-use, I'm gonna leave that to a follow up exercise :D I think there's a lot more re-use to be had in this file.